### PR TITLE
feat: add vale linter

### PR DESCRIPTION
**Pull request by tthsympa**
##  The what
The goal is to add [Vale](https://docs.errata.ai/vale/about) prose-linter.

We need to:

- [x] Make sure the tool is what MTA team wants
- [ ] Add more vocabulary into the tool (scaleway-specific words)
- [ ] Add a [GitHub action](https://github.com/errata-ai/vale-action) ?

EDIT: please see https://github.com/scaleway/docs-content/pull/66#issuecomment-912491372 for the last two tasks

## To test
- `brew install vale`
- Make sure the command is well installed -> `vale -h`
- Try the command on a file or folders
`vale ai/index.mdx` -> no errors
`vale ai/` -> errors in `ai/concepts.mdx`
- You can also try [VSCode extension](https://github.com/errata-ai/vale-vscode)

### CODE REVIEW

File: styles/Microsoft/Acronyms.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+message: "'%s' has no definition."
+link: https://docs.microsoft.com/en-us/style-guide/acronyms
+
+level: suggestion
+ignorecase: false
+# Ensures that the existence of 'first' implies the existence of 'second'.
+first: '\b([A-Z]{3,5})\b'
+second: '(?:\b[A-Z][a-z]+ )+\(([A-Z]{3,5})\)'
+# ... with the exception of these:
+exceptions:
```
Comment (kindermoumoute) [inline styles/Microsoft/Acronyms.yml:11]: A lot of Scaleway-specific acronyms are missing from this list. I suggest you sync it with the one used in the Go SDK: https://github.com/scaleway/scaleway-sdk-go/blob/master/strcase/goname.go#L102-L142

Comment (tthsympa) [reply styles/Microsoft/Acronyms.yml:11]: I also started a list [here](https://github.com/scaleway/docs-content/pull/66/files?file-filters%5B%5D=.json&file-filters%5B%5D=.md&file-filters%5B%5D=.txt&file-filters%5B%5D=.yml#diff-747d026a1d56651b86d06324c386e8ec49655b5133665a6a2a5546472d06fc00)

**Comment by tthsympa** [comment]
We will merge this PR as is. This will be a background task for the MTA team to populate the Vocab/Acronyms files.

File: styles/Microsoft/DateFormat.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+extends: existence
```
Comment (ldecarvalho-doc) [inline styles/Microsoft/DateFormat.yml:1]: Je pense qu'on peut effacer ce style

File: styles/Microsoft/DateOrder.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+extends: existence
```
Comment (ldecarvalho-doc) [inline styles/Microsoft/DateOrder.yml:1]: on peut effacer celui-la aussi.

**Review by ldecarvalho-doc — APPROVED** [review]

File: styles/Microsoft/DateFormat.yml (hunk 2 above)
Comment (tthsympa) [reply styles/Microsoft/DateFormat.yml]: Done

File: styles/Microsoft/DateOrder.yml (hunk 3 above)
Comment (tthsympa) [reply styles/Microsoft/DateOrder.yml]: Done

**Review by ofranc — APPROVED** [review]

**Pull request merged**